linera-views/src/key_value_store_view.rs

```diff
@@ -52,12 +52,12 @@ enum KeyTag {
 /// The updates and deleted_prefixes have to be coherent. This means:
 /// * If an index is deleted by one in deleted_prefixes then it should not be present
 ///   in updates at al.
-/// * deleted prefix in deleteprefix should not dominate anyone. That is if
+/// * deleted_prefix in DeletePrefix should not dominate anyone. That is if
```

Comment:
```suggestion
/// * `key_prefix` in `DeletePrefix` should not dominate anyone. That is if
```

The field of the `DeletePrefix` variant is what you're referring to, is it?

Reply:
Yes.

Reply:
How about
```
[`DeletePrefix::key_prefix`]
```
?